pom.xml

```diff
@@ -257,5 +122,17 @@
 			<organizationUrl>https://marvelution.com</organizationUrl>
 			<timezone>+1</timezone>
 		</developer>
+    <developer>
+      <name>Eugen Mayer</name>
+      <organization>KontextWork</organization>
+      <organizationUrl>https://kontextwork.de</organizationUrl>
+      <timezone>+1</timezone>
+    </developer>
 	</developers>
+
+  <scm>
```

Comment:
For my education; Do you have a specific reason for moving the scm element to the end of the file.

Reply:
No not at all, i assume it was some kind of preference like "the things you need for secondary maintenance like dep-management/properties as far top up, things you do not change regularly like repos/scm' far down. There was no other rational here - if it was too ballsy, let me know, i try to place it back were it was.